package.json

```diff
@@ -677,9 +677,15 @@
           "runme.experiments.grpcSerializer": {
             "type": "boolean",
             "scope": "window",
-            "default": true,
+            "default": false,
             "markdownDescription": "If set to `true`, the notebook serialization will use the GRPC instead of the WebAssembly interface."
           },
+          "runme.experiments.connectSerializer": {
```

Comment:
Can we make this a string serializerAddress?
https://docs.google.com/document/d/1blm5kD0rb0X6hyb3w40gwYrhKzeCVbxGwLJUb0vkCCo/edit#bookmark=id.tdj09kndh2f6